packages/rocketchat-api/server/v1/users.js

```diff
@@ -363,6 +363,13 @@ RocketChat.API.v1.addRoute('users.setPreferences', { authRequired: true }, {
 			preferences = _.extend({ _id: userId, settings: { preferences: this.bodyParams.data } });
 		}
 
+		// Keep compatibility with old values
+		if (preferences.emailNotificationMode === 'all') {
```

Comment:
But sometimes we do need '`all'`, not just `'mentions'`, for example, for people who don't have RocketChat client, and we want to use e-mail as notifications, for important channels. People should be able to get notifications when they are not on-line.

Reply:
It would be so great to have email notifications for all messages due to push-notifications issues on some rocket.chat instances, like on wechange.de. With this I could create a mail-filter and forward all mails to IFTTT, that triggers a push notification on the phone. - It would be even greater to have an option to change the mail content for email-notifications to `minimal`, where mail body holds just the deep link to the rocket.chat conversation and the subject holds the channel title (with some prefix, like the rocket.chat domain or something like that).
With this it would be quite easy to set up a IFTTT rule for getting push-notifications per user via email notifications without having the rocket.chat host to set up its own relay for push notifications. And users could get up to 750 notifications for free with the free tier of IFTTT.